k8s/installation/observability.mdx

```diff
@@ -0,0 +1,68 @@
+---
+title: Monitoring the Kubernetes Operator
+sidebarTitle: Observability
+description: Learn how to enable and customize ngrok's Kubernetes ingress Observability features.
+---
+
+Configuring metrics and logging for the ngrok Kubernetes Operator is essential for gaining visibility into how your ngrok ingress resources behave within your cluster. This guide explains the available observability features, how to enable and how to customize them. By setting up proper metrics and logs, you’ll be able to diagnose issues more quickly, track system performance, and ensure the operator is running reliably in production.
```

Comment:
```suggestion
Configuring metrics and logging for the ngrok Kubernetes Operator is essential for gaining visibility into how your ngrok ingress resources behave within your cluster.

This guide explains the available observability features, and how to enable and customize them to:
- Diagnose issues more quickly.
- Track system performance.
- Ensure the operator is running reliably in production.
```

I think it's a bit easier to read and or skim the info if it's broken out like this